### PR TITLE
Changed inventoryUpdate category dropdown to default to product's cur…

### PR DESCRIPTION
**Pull request by nramey3**
…rent category by editing bootstable.js.

### CODE REVIEW

File: public/assets/bootstrap/js/bootstable.js [hunk 1]
```diff
@@ -208,6 +208,7 @@ function butRowEdit(but) {
     //Start the edition mode for a row.
     var $row = $(but).parents('tr');  //accede a la fila
     var $cols = $row.find('td');  //lee campos
+    var $category = $row.find('td:eq(3)').html();
```
Comment (bpawletzki) [inline public/assets/bootstrap/js/bootstable.js]: can 3 be idx

`var $category = $row.find('td:eq(3)').html();`
`var $category = $row.find('td:eq(idx)').html();`

Comment (bpawletzki) [reply public/assets/bootstrap/js/bootstable.js]: var $category can be removed. The var cont contains what you need to do the compare.

File: public/assets/bootstrap/js/bootstable.js [hunk 2]
```diff
@@ -236,7 +237,11 @@ function butRowEdit(but) {
                 input = '<select class="form-control input-sm" >';
 
                 opts.forEach(function (op) {
-                    input += '<option>' + op + '</option>';
+                    if(op == $category) {
```
Comment (bpawletzki) [inline public/assets/bootstrap/js/bootstable.js]: Change $category to cont.
`                    if(op == cont) {`

Comment (nramey3) [reply public/assets/bootstrap/js/bootstable.js]: Done

**Review by bpawletzki — APPROVED** [review]

**Pull request merged**